CMakeLists.txt

```diff
@@ -269,11 +287,13 @@ endif()
 install(TARGETS libdebug_ptrace_binding LIBRARY DESTINATION libdebug/ptrace/native)
 install(TARGETS libdebug_linux_binding LIBRARY DESTINATION libdebug/native)
 install(TARGETS libdebug_debug_sym_parser LIBRARY DESTINATION libdebug/native)
+install(TARGETS libdebug_elf_api LIBRARY DESTINATION libdebug/native)
 
 # Install the Python stubs
 install(FILES ${CMAKE_BINARY_DIR}/libdebug_ptrace_binding.pyi DESTINATION libdebug/ptrace/native)
 install(FILES ${CMAKE_BINARY_DIR}/libdebug_linux_binding.pyi DESTINATION libdebug/native)
 install(FILES ${CMAKE_BINARY_DIR}/libdebug_debug_sym_parser.pyi DESTINATION libdebug/native)
+install(FILES ${CMAKE_BINARY_DIR}/libdebug_elf_api.pyi DESTINATION libdebug/native)
```

Comment:
This is a reminder for me and not a review comment, but I have to re-check this stub installation part as I'm not sure that this is the correct way to handle them